### DISCUSSION TITLE
File naming convention specifies .json file extension

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#56] [Q&A] [answered]
The file naming convention on the readme.md page specifies a .json file extension, I assume this is just an example and other extensions would be used for other valid data file types.  I think this should be clarified in the document.

**Comment by shaselton-usds** [comment, marked as answer]
Thanks for the catch. You're definitely correct -- clarification has been added: https://github.com/CMSgov/price-transparency-guide/commit/2f51f38e584ed1c5d54d3718b83a060f34a1d039
